src/contrib/mlir/compiler/lowerer.cpp

```diff
@@ -804,11 +794,13 @@ namespace
         //           Output[n, k, r_1, .. r_f] +=
         //             Images[n, c, i_1 + j_1, .. i_f + j_f] * Filters[k, c, j_1, .. j_f]
 
-        // TODO: With padding, we need to check (using IntegerSets) whether each spatial dim in
-        // Images lie within paddings
-        // If yes, we init value to zero, else load from MemRef.
-        // Q: Can this be done using a map from padded tensor to  unpadded one ? Will we load zero
-        // if OOB ?
+        // With padding, we check (using IntegerSets) whether each spatial dim in Images lie within
+        // paddings. If true, we perform the computation:
```

Comment:
"lie within paddings" sounds as if we performed computation for elements in the padding. Should we say outside of padding instead?